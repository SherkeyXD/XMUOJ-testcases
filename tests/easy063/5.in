9956
2040
6546
9576
295
3400
6110
3776
7040
5201
9703
50
2535
6162
7780
6052
9695
1464
2780
6386
8383
2545
9425
4520
5624
523
3987
9228
9025
205
9756
944
6036
6980
4360
9875
4880
6660
9704
9500
432
7878
9480
6020
6860
5820
7520
9031
1300
3610
5499
1973
2824
1346
9215
3938
8132
7250
8675
6528
9924
880
7310
6096
8120
8939
6388
5728
3720
6492
5304
9642
1391
3294
9500
6879
5120
1852
3800
5784
8960
7744
4700
9268
6313
8970
9388
3882
6725
2939
7185
8363
6668
9414
7336
1528
8210
360
3696
1792
9218
4098
3950
2023
8256
7328
6080
8902
413
568
3550
2172
3080
4076
8
1600
7680
3825
7135
1390
3341
8259
5400
4905
870
9060
8333
8205
8281
6340
1050
9500
6929
180
6154
2575
5828
9630
7980
2600
1872
9778
2424
5388
3000
1545
2830
3320
992
7424
843
6152
8272
5516
8480
1848
5280
1960
8996
8981
5468
6562
8120
2336
8496
6439
4821
8488
7451
4680
2191
1076
5539
293
9872
7060
2615
7136
7382
3960
7200
8168
8710
16
2600
4096
8795
350
9888
312
8240
3364
955
4598
200
1555
8912
3098
4546
1185
5167
1724
9910
2187
7305
1014
64
2440
3536
9740
5286
7649
8132
4980
7970
6440
1927
4512
2306
6468
3803
2448
3305
4775
8805
0
