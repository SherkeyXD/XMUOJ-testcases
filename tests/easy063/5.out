2
6
3
9
7
7
8
5
2
8
1
5
6
6
4
4
2
6
8
5
4
7
2
2
8
1
9
3
7
7
9
8
6
5
4
2
2
9
2
5
9
3
3
8
2
6
5
4
4
1
9
2
7
5
8
5
5
5
8
3
6
7
2
3
2
2
7
4
3
3
3
3
5
9
5
3
8
7
2
6
5
4
2
7
4
6
1
3
2
5
3
2
8
9
1
7
2
9
6
1
2
3
8
7
3
2
5
1
8
1
4
3
2
8
8
7
3
9
7
4
2
6
9
9
6
6
8
6
1
4
6
5
8
9
7
1
5
9
6
8
9
4
3
6
3
6
4
8
2
8
6
5
1
8
2
3
6
7
5
8
5
1
2
5
9
4
6
1
8
9
4
5
4
5
8
4
5
8
2
9
9
5
7
7
8
1
2
8
6
6
5
7
1
8
2
7
2
2
1
6
1
5
1
9
6
6
1
1
8
2
3
8
5
3
5
5
1
3
2
6
5
9
2
5
3
